pkg/cluster/driver/oci/capi.go

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) 2024, 2025, Oracle and/or its affiliates.
 // Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
 
-package capi
+package oci
 
```

Comment:
I think this should be driver.go, not capi.go

Reply:
fixed